solidity/contracts/bridge/Bridge.sol

```diff
@@ -1385,18 +1399,35 @@ contract Bridge is Ownable {
                     // key from the mapping to make it reusable for further
                     // requests.
                     delete pendingRedemptions[redemptionKey];
+
+                    processedRedemptionsCount++;
```

Comment:
Could be cheaper to set a bool to true, but I am not sure. Something to investigate later.

Reply:
We do it in the same way in `submitSweepProof` so I wanted to be consistent here.

Reply:
Can we leave a TODO here to revisit if this should not be just a boolean in terms of gas efficiency? Same for the sweep flow.

Reply:
Added it as part of: https://github.com/keep-network/tbtc-v2/pull/128/commits/deef0f3a97fe4ede455bfb6a0da60e9a419c8799

Reply:
For sweep flow we need `processedDepositsCount` anyway so I think it's not the case there.